src/routes/ConfigureQuiz/QuestionCard.tsx

```diff
@@ -92,7 +93,7 @@ export default function QuestionCard({
     updateQuestionData(idx, question);
     await updateQuizData();
     setActiveQuestionIndex(null);
-    setExpandedQuestionIndex(null);
```

Comment:
why this change ?

Reply:
Want to keep the question expanded after edit so that they can preview the saved question as we don't have preview mode anymore.

Reply:
When they one more question or expand another question then that one will be collapsed automatically.